zebedee-cms/src/test/java/com/github/onsdigital/zebedee/api/ContentTest.java

```diff
@@ -143,17 +158,25 @@ public void writeVersionFileV1() throws Exception {
         ZebedeeCollectionReaderFactory factory = new ZebedeeCollectionReaderFactory(mockZebedee);
         ZebedeeReader.setCollectionReaderFactory(factory);
 
+        ByteArrayOutputStream content = new ByteArrayOutputStream(1024);
+        when(mockResponse.getOutputStream())
+                .thenReturn(new StubServletOutputStream(content));
+
+
         try {
             Data dataAPI = new Data();
-            dataAPI.read(dataRequest, response);
+            dataAPI.read(dataRequest, mockResponse);
         } catch (Exception e) {
             e.printStackTrace();
             Assert.fail(e.getMessage() + e.toString());
             throw e;
         }
 
-        Assert.assertEquals(200, response.getStatus());
-        JSONAssert.assertEquals(EXPECTED_CONTENT_V1, response.getContentAsString(), false);
+        // admittedly a little fragile - ideally we'd like to be sure that the last invocation (or possibly default value) was correct.
+        verify(mockResponse, atLeast(1)).setStatus(HttpStatus.SC_OK);
+        verify(mockResponse, atLeast(1)).setContentType(MediaType.APPLICATION_JSON);
```

Comment:
Is using `atLeast` what makes this a little fragile? Should we use `times` instead?